easybuild/easyblocks/m/molpro.py

```diff
@@ -67,85 +88,95 @@ def configure_step(self):
                     raise EasyBuildError("Failed to create symlink for license token at %s", self.license_token)
 
             else:
-                # no license token available, so fail early
-                raise EasyBuildError("No license token found at either %s or via 'license_file'", self.license_token)
+                self.log.warning("No licence token found at either {0} or via 'license_file'".format(self.license_token))
```

Comment:
This change was made to accommodate cases in which one installation of Molpro might serve several users, each of which brings his own licence token to the table, so to speak. Requiring a token at installation time (when the HPC facility might not have its own Molpro licence but is installing on behalf of users) breaks this approach.